pythoncalculator/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .add import add 
-
+from .multiply import multiply 
```

Comment:
The checks say they failed, does this need to be on separate lines, e.g., 

from .multiply 
import multiply

?

Reply:
Can you see below it says

"All checks have failed?"

Reply:
No, these need to be on the same line, it is saying "From the module `multiply` import the function multiply" and so it should all be one line.